cfgov/v1/migrations/9999_init_data.py

```diff
@@ -15,24 +14,28 @@
 
 def initial_data(apps, schema_editor):
     if settings.DEBUG:
-        admin_user = User(username='admin',
-                          password=make_password(os.environ.get('ADMIN_PW')),
-                          is_superuser=True, is_active=True, is_staff=True)
-        admin_user.save()
+        UserModel = apps.get_model(settings.AUTH_USER_MODEL)
+        if not UserModel.objects.filter(username='admin').exists():
+            admin_user = UserModel(username='admin',
+                              password=make_password(os.environ.get('WAGTAIL_ADMIN_PW')),
+                              is_superuser=True, is_active=True, is_staff=True)
+
+            admin_user.save()
 
         # Renaming Site root name to `V1`
-        site_root = Page.objects.get(id=2)
-        site_root.title = 'V1'
-        site_root.save()
+        if not Page.objects.filter(title='V1').exists():
+            site_root = Page.objects.get(id=2)
+            site_root.title = 'V1'
+            site_root.save()
 
         # Events Landing Page required for event `import-data` command
-        events = EventLandingPage(title='Events', slug='events')
-        site_root.add_child(instance=events)
-        revision = events.save_revision(
-            user=admin_user,
-            submitted_for_moderation=False,
-        )
-        revision.publish()
+        if not EventLandingPage.objects.filter(title='Events').exists():
```

Comment:
The `EventLandingPage` type is deprecated in the BrowseFilterablePage PR. It's a normal `BrowsePage` now.